ui/src/components/about/OurHeartMissions.js

```diff
@@ -0,0 +1,206 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  Stack,
+  VStack,
+  Text,
+  Container,
+  HStack,
+} from '@chakra-ui/react';
+
+const Picture = ({ image, year, text }) => {
+  return (
+    <VStack h="100%" px={2} py={3}>
+      <Box
+        w="100%"
+        h="7em"
+        bgImage={image}
+        bgPos="center"
+        bgSize="contain"
+        bgRepeat="no-repeat"
+      ></Box>
+      <Text
+        color="black"
+        fontWeight={600}
+        textAlign="center"
+        px={{ base: 5, sm: 4, md: 7, lg: 7 }}
+      >
+        {year}
+      </Text>
+      <Text
+        color="black"
+        fontFamily="Manrope"
+        fontWeight={400}
+        textAlign="center"
+        fontSize={{ base: 'smaller', md: 'md' }}
+      >
+        {text}
+      </Text>
+    </VStack>
+  );
+};
+
+const MissionSection = (props) => {
+  const { title, blurb } = props;
+  return (
+    <Flex maxW="container.lg">
+      <VStack justifyContent="space-around" spacing="2em">
+        <Box>
+          <Heading
+            textAlign="center"
+            textColor="rgba(66, 86, 158, 1)"
+            fontSize={['xl', '3xl']}
+            fontWeight={700}
+            paddingBottom={2}
+          >
+            {title}
+          </Heading>
+          <Text textAlign="center" px="1em">
+            {blurb.blurb[1]}
+          </Text>
+        </Box>
+        <Container backgroundColor="rgba(238, 250, 255, 1)" maxW="container.lg">
+          <Box backgroundColor="rgba(238, 250, 255, 1)" px={{ base: 1, md: 3 }}>
+            <Heading
+              textAlign="left"
+              textColor="rgba(66, 86, 158, 1)"
+              fontSize={['xl', '3xl']}
+              fontWeight={700}
+              paddingTop="1em"
+            >
+              {blurb.title[2]}
+            </Heading>
+            <Text paddingBottom="2em" textAlign="justify">
+              {blurb.blurb[2]}
+            </Text>
+          </Box>
+        </Container>
+        <Container backgroundColor="rgba(243, 243, 243, 1)" maxW="container.lg">
+          <Box px={{ base: 1, md: 3 }}>
+            <Heading
+              textAlign="left"
+              textColor="rgba(66, 86, 158, 1)"
+              fontSize={['xl', '3xl']}
+              fontWeight={700}
+              py="1em"
+            >
+              {blurb.title[3]}
+            </Heading>
+            <Text fontWeight={700} fontSize={['md', 'xl']}>
+              {blurb.heading[1]}
+            </Text>
+            <Text paddingBottom="2em" textAlign="justify">
+              {blurb.blurb[3]}
+            </Text>
+            <Text>{blurb.blurb[4]}</Text>
+          </Box>
+          {}
+          <Stack
```

Comment:
<img width="499" alt="image" src="https://user-images.githubusercontent.com/24607458/185398469-d7c87603-eb1a-45b2-8803-58c60b1141af.png">

text fontfamily seems wrong in my browser. I used chrome and brave.